server/liveblog/embed/embed.py

```diff
@@ -11,12 +11,28 @@
 """Embed module"""
 import superdesk
 from flask import render_template, request
+from superdesk import get_resource_service
 
-bp = superdesk.Blueprint('embed_liveblog', __name__, template_folder='templates', static_folder='assets')
+ASSETS_DIR = 'embed_assets'
+bp = superdesk.Blueprint('embed_liveblog', __name__, template_folder='templates', static_folder=ASSETS_DIR)
 
 
 @bp.route('/embed/<blog_id>')
 def embed(blog_id):
-    return render_template('embed.html', blog_id=blog_id, api_host=request.url_root)
+
+    def complete_url(url):
+        def is_relative(url):
+            return not (url.startswith('/') or url.startswith('http://') or url.startswith('https://'))
```

Comment:
This function is used to know if the path in the url is relative or not.
Is relative:
 - script.js
- script/main.js
Is not relative:
- /script/main.js
- //maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css
- https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css

Any way, I think we will need to exclude remote files (starting with the protocol or `//`) because we don't want to have to many `<script>` tags in the final embed html then we need to compile and minify them